server/chat.ts

```diff
@@ -1094,27 +1099,41 @@ export class CommandContext extends MessageContext {
 						this.errorReply(`Images are not allowed in personal rooms.`);
 						return null;
 					}
-					if (!/width=([0-9]+|"[0-9]+")/i.test(tagContent) || !/height=([0-9]+|"[0-9]+")/i.test(tagContent)) {
+					if (!/width ?= ?(?:[0-9]+|"[0-9]+")/i.test(tagContent) || !/height ?= ?(?:[0-9]+|"[0-9]+")/i.test(tagContent)) {
 						// Width and height are required because most browsers insert the
 						// <img> element before width and height are known, and when the
 						// image is loaded, this changes the height of the chat area, which
 						// messes up autoscrolling.
 						this.errorReply(`All images must have a width and height attribute`);
+						this.errorReply(`This image doesn't: <${tagContent}>`);
```

Comment:
```suggestion
						this.errorReply(`This image is missing a width/height attribute: <${tagContent}>`);
						this.errorReply(`Images without predefined width/height cause problems with scrolling because loading them changes their height.`);
```